src/public/app/services/load_results.ts

```diff
@@ -208,6 +206,7 @@ export default class LoadResults {
     }
 
     getAttachmentRows() {
+        console.log("getATttt", this.attachmentRows)
```

Comment:
Leftover log.

Reply:
oops :-D
will fix it, thanks for noticing
(and note to self: check diffs more thoroughlyl in the future ;-))